client/src/com/aerospike/client/AerospikeClient.java

```diff
@@ -2013,16 +2132,45 @@ public final IndexTask dropIndex(
 		}
 		sb.append(";indexname=");
 		sb.append(indexName);
+		return sb.toString();
+	}
 
-		// Send index command to one node. That node will distribute the command to other nodes.
-		String response = sendInfoCommand(policy, sb.toString());
+	/**
```

Comment:
Any chances you could split this PR into small ones? It is really difficult to review the changes introduced